superbench/benchmarks/micro_benchmarks/gemm_flops_performance.py

```diff
@@ -0,0 +1,171 @@
+            'FP16_TC': 'cutlass_tensorop_h16816gemm_256x128_32x3_*',
+            'INT8_TC': 'cutlass_tensorop_s8_i16832gemm_s8_256x128_64x3_*',
+            'INT4_TC': 'cutlass_tensorop_s4_i16864gemm_s4_256x128_128x3_*',
+        }
+
+    def add_parser_arguments(self):
+        """Add the specified arguments."""
+        super().add_parser_arguments()
+
+        self._parser.add_argument(
+            '--num_warmup',
+            type=int,
+            default=5,
+            required=False,
+            help='The number of warmup step.',
+        )
+        self._parser.add_argument(
+            '--n',
+            type=int,
+            default=16384,
+            required=False,
+            help='The N dim of matmul (N, K) * (K, M).',
+        )
+        self._parser.add_argument(
+            '--k',
+            type=int,
+            default=16384,
+            required=False,
+            help='The K dim of matmul (N, K) * (K, M).',
+        )
+        self._parser.add_argument(
+            '--m',
+            type=int,
+            default=16384,
+            required=False,
+            help='The M dim of matmul (N, K) * (K, M).',
+        )
+        self._parser.add_argument(
+            '--precision',
+            type=str,
+            nargs='+',
+            default=list(self.__kernel_map.keys()),
+            help='Precision for benchmarking. E.g. {}.'.format(' '.join(list(self.__kernel_map.keys()))),
+        )
+
+    def _preprocess(self):
+        """Preprocess/preparation operations before the benchmarking.
+
+        Return:
+            True if _preprocess() succeed.
+        """
+        if not super()._preprocess():
+            return False
+
+        self._args.precision = [p.upper() for p in self._args.precision]
+        for p in self._args.precision:
+            if p not in list(self.__kernel_map.keys()):
+                self._result.set_return_code(ReturnCode.INVALID_ARGUMENT)
+                logger.error(
+                    'Unsupported precision - benchmark: {}, precision: {}, expected: {}.'.format(
+                        self._name, p, list(self.__kernel_map.keys())
+                    )
+                )
+                return False
+            else:
+                command = os.path.join(self._args.bin_dir, self._bin_name)
+                command += (' --warmup-iterations=' + str(self._args.num_warmup))
+                command += (' --operation=gemm')
+                command += (' --n=' + str(self._args.n))
+                command += (' --k=' + str(self._args.k))
+                command += (' --m=' + str(self._args.m))
+                command += (' --kernels=' + self.__kernel_map[p])
+                self._commands.append(command)
+
+        # TODO - To support more architecutres, currently only support compute capability = 7.0 or 8.0
+        capability = nv_helper.get_device_compute_capability()
+        if capability == 7.0:
```

Comment:
```suggestion
        if torch.cuda.get_device_capability() == (7, 0):
```

Reply:
It's a little strange that just import torch for one cuda-related function call.